lib/components/program-form/index.tsx

```diff
@@ -0,0 +1,46 @@
+import { FormControl, FormLabel, Input, Textarea, Flex, Checkbox } from '@chakra-ui/react';
+
+type ProgramFormProps = {};
```

Comment:
Do we need to include this if it doesn't take in any props?

Reply:
good catch!